consoles/sshXtermVt.pm

```diff
@@ -35,6 +36,17 @@ sub activate {
     my $ssh_args        = $self->{args};
     my $gui             = $self->{args}->{gui};
 
+    # Wait that SUT is live on network (for generalhw/ssh)
+    my $p       = Net::Ping->new();
```

Comment:
I had something similar in my mind. And I would prefer to probe ssh port directly like: https://github.com/cfconrad/os-autoinst/commit/4cc23ac953ea2cd83dfbd6e385f2d57b25f8c591

Reply:
I am generally ok with your solution (not tested yet), even if I would prefer a bigger default wait value, but this must be configurable by a `get_var` as done here.
I can rebase my PR on top of your changes, once they will be merged.

Reply:
I didn't created a PR for it yet. As I only saw it once during my local tests and so I wasn't sure if it is really needed. Feel free to grap it.

Reply:
Ok, so I grabbed your code and reworked it a bit.
Thanks!